src/action.ts

```diff
@@ -8,37 +8,44 @@
 
 'use strict';
 
-const utils = require('./utils');
+import { Device } from './device';
+import { timestamp } from './utils';
+
+export interface ActionDescription {
+  id: string
+  name: string
+  input: any,
```

Comment:
```suggestion
  input: any
```

Also, you use semicolons in other palces so maybe for interface as well?

Reply:
I don't have any preference.
We could also omit the commas or use semicolons.
What do you think @mrstegeman?

Reply:
I tend to prefer explicit semicolons.

Reply:
BTW, those things can be autofixed by `npm run lint -- --fix` once it is set up for TypeScript, so I wouldn't put much effort into this right now.